test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -60,27 +59,33 @@ public class CommandRunner
 
             Process p = null;
 
-            try
+            using (p = new Process())
```

Comment:
Can you shortly elaborate in more detail how your change helps? 

I know you linked the issues and I appreciate that, but I was hoping you could save us some time digging through the linked docs if possible.

Reply:
Updated PR description with additional information. Pls let me know your feedback.